interface/main/tabs/main.php

```diff
@@ -105,15 +105,17 @@ function isEncounterLocked( encounterId ) {
 var webroot_url="<?php echo $web_root; ?>";
 </script>
```

Comment:
I separated out this improvement in the tabs layout gui here:
https://github.com/openemr/openemr/pull/763